data-service/src/main/java/uk/gov/gchq/palisade/service/data/config/StdSerialiserPrepopulationFactory.java

```diff
@@ -104,20 +93,28 @@ public void setSerialiserClass(final String serialiserClass) {
      *
      * @return an {@link Entry} that consists of the created {@link DataFlavour} and {@link Serialiser} objects.
      */
+    @SuppressWarnings("unchecked")
     public Entry<DataFlavour, Serialiser<Object>> build() {
         Serialiser<Object> serialiser;
+        LOGGER.info("Building serialiser for class '{}', resource format '{}' and domain type '{}'", serialiserClass, flavourFormat, flavourType);
         try {
-            serialiser = (Serialiser<Object>) Class.forName(serialiserClass)
-                    .getConstructor(Class.class)
-                    .newInstance(Class.forName(flavourType));
+            Class<?> serialiserClazz = Class.forName(serialiserClass);
+            LOGGER.debug("Got class {} for serialiser classname {}", serialiserClazz, serialiserClass);
+            Constructor<?> serialiserConstructor = serialiserClazz.getConstructor(Class.class);
+            LOGGER.debug("Got one-element constructor {} for class {}", serialiserConstructor, serialiserClazz);
+            Class<?> typeClass = Class.forName(flavourType);
+            LOGGER.debug("Got class {} for flavourType classname {}", typeClass, flavourType);
```

Comment:
Do we really need all these logging messages for this process? It seems like a lot of noise.

Reply:
Changed to trace